felix/ifacemonitor/iface_monitor.go

```diff
@@ -219,6 +219,11 @@ func (m *InterfaceMonitor) handleNetlinkRouteUpdate(update netlink.RouteUpdate)
 		return
 	}
 
+	// issue: https://github.com/projectcalico/calico/issues/7645
```

Comment:
Can you please remove this comment. By using `git blame` and checking the PR, it's possible to get back to the original issue.

Reply:
Thanks for the review @mazdakn , Now updated.